gstreamer_proxy:
  ros__parameters:
    stream_address: rtsp://192.168.1.17:8554/barlus
    distortion_model: plumb_bob
    frame:
      height: 2048
      width: 3072
    camera_matrix: [1.0, 0.0, 0.0,
                    0.0, 1.0, 0.0,
                    0.0, 0.0, 1.0]
    distortion_coefficients: [0.0, 0.0, 0.0, 0.0, 0.0]
    projection_matrix: [1.0, 0.0, 0.0, 0.0,
                        0.0, 1.0, 0.0, 0.0,
                        0.0, 0.0, 1.0, 0.0]
